Tests/OMTestCase.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace WebtownPhp\BannerBundle\Tests;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Doctrine\ORM\Tools\SchemaTool;
+use Nelmio\Alice\Fixtures;
+use Symfony\Bridge\Doctrine\Test\DoctrineTestHelper;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use WebtownPhp\BannerBundle\Entity\BannerRepository;
+use WebtownPhp\BannerBundle\Manager\ManagerInterface;
+use WebtownPhp\BannerBundle\Manager\ORMManager;
+
+/**
+ * Test case with DB access
+ *
+ * @author Zoltan Feher <whitezo@webtown.hu>
+ */
+class OMTestCase extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var ObjectManager
+     */
+    protected $em;
+
+    /**
+     * @var ManagerInterface
+     */
+    protected $bm;
+
+    /**
+     * Test setup
+     */
+    protected function setUp()
+    {
+        // setup entity aliases
+        $this->em = DoctrineTestHelper::createTestEntityManager();
+        $entityManagerNamespaces = $this->em->getConfiguration()->getEntityNamespaces();
+        $entityManagerNamespaces['WebtownPhpBannerBundle'] = 'WebtownPhp\BannerBundle\Entity';
+        $this->em->getConfiguration()->setEntityNamespaces($entityManagerNamespaces);
+
+        // setup schema
+        $schemaTool = new SchemaTool($this->em);
+        $classes = [];
+        foreach ($this->getEntities() as $entityClass) {
+            $classes[] = $this->em->getClassMetadata($entityClass);
+        }
+        try {
+            $schemaTool->dropSchema($classes);
+        } catch (\Exception $e) {
```

Comment:
Biztosan jó ötlet ezeket a hibákat elnyelni? Nem lehet úgy megoldani, hogy hiba nélkül lefusson és csak akkor dobjon exception-t ha tényleg vmi gubanc van, akkor viszont azt nem nyeljük el?
Az a baj az ilyen elnyelt hibákkal, hogy néha elnyelnek olyan problémákat is, amiket nem kellene, az ember meg nem érti, hogy minden oké, mégsem oké semmi, és nem kap hibaüzenetet, vagy vmi olyat kap, aminek nincs értelme.